app/build.gradle

```diff
@@ -124,6 +124,8 @@ dependencies {
     implementation 'com.google.guava:guava:28.1-jre'
     annotationProcessor 'com.google.guava:guava:24.1-jre'
     implementation 'org.apache.commons:commons-lang3:3.9'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
+    implementation 'com.makeramen:roundedimageview:2.3.0'
```

Comment:
Also this library is not kept up to date and there is a bunch of issues on it..

Is there no alternative ?